src/elements/common/nav-router/NavRouter.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { MemoryRouter, MemoryRouterProps, Router } from 'react-router';
+import { History } from 'history';
+
+type Props = {
+    children: React.ReactNode;
+    history?: History;
+} & MemoryRouterProps;
```

Comment:
This isn't present on the current component. Will it require developers to pass in a router and/or its props? It's meant to be an optional feature. Will it work with non-memory routers?

Reply:
Yeah I think due to flow, it’s not restricted but ContentSidebar makes use of the “hidden” props for the MemoryRouter. They’re optional anyway so users don’t have to pass anything in. I think this is just a more accurate typing

Reply:
I see. Does this work with non-memory routers? If I pass in a BrowserRouter, will Typescript complain?